app/src/main/kotlin/com/bitwarden/authenticator/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -102,6 +104,18 @@ class SettingsDiskSourceImpl(
             mutableFirstLaunchFlow.tryEmit(hasSeenWelcomeTutorial)
         }
 
+    override var previouslySyncedBitwardenAccountIds: Set<String>
+        get() = getStringSet(
+            key = PREVIOUSLY_SYNCED_BITWARDEN_ACCOUNT_IDS_KEY,
+            default = emptySet(),
+        ) ?: emptySet()
```

Comment:
Tests here and below, if we can.

Reply:
Doh, thanks for the reminder! Love me some simple and effective disk source tests:

https://github.com/bitwarden/authenticator-android/pull/259/commits/2588d7fdea55fadcc9252c4a20f1e466dc0ab990